modules/virt-hot-plugging-memory.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-hot-plugging-memory_{context}"]
+
+= Hot plugging memory on a virtual machine
+
+You can add or remove the amount of memory allocated to a virtual machine (VM) without having to restart the VM by using the {product-title} web console or editing the YAML file.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines*.
+. Select the required VM to open the *VirtualMachine details* page.
+. On the *Configuration* tab, click *Edit CPU|Memory*.
+. Enter the desired amount of memory and click *Save*.
+
+The system applies these changes immediately. If the VM is migratable, a live migration is triggered. If not, or if the changes cannot be live-updated, a `RestartRequired` condition is added to the VM.
+
+[NOTE]
+====
+Linux guests require a kernel version of 5.16 or higher and Windows guests require the latest `viomem` drivers.

```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#references-to-earlier-or-later-versions) this should be "or later":
```suggestion
Linux guests require a kernel version of 5.16 or later and Windows guests require the latest `viomem` drivers.

```